site/app/Resources/views/base.html.twig

```diff
@@ -155,6 +148,8 @@
       <script src="{{ asset_url }}"></script>
     {% endjavascripts %}
 
+    {{ encore_entry_script_tags('app') }}
```

Comment:
What happens to the JS files to be included at the end of the file? (i.e. most of them?)

Reply:
I think we add js files at the end because we don't have to worry if the DOM has been loaded or not, everything will work fine.

Reply:
We add JS files at the end because it's good for performance. We only add JS files to the header which have need to be there. This current setup should be preserved after we move to webpack.